### DISCUSSION TITLE
6 column build w/ cluster like a Kinesis 360?

### DISCUSSION THREAD

**Discussion by pjeroff** [juhakaup/keyboards#46] [Ideas]
Heya! 

Building my first DIY keeb for my brother, and since I have the 360 from Kinesis I've landed upon the Dactyl-CC DIY one. 
Now that I'm wiring it (after a laborious print....with Nylon-CF of all things...) I have come to realize the design is bad, as it doesn't allow for the Amoeba's PCB's to fit well (or at all in certain corners)...as well as there are no good brackets for the RP2040-like controllers like you've made.

Would it be hard to do your design, but in 6column + different thumb-cluster? Is there a video to showcase how you did your design, or where to learn the necessary skills to update your brilliant design?

Cheers!

**Comment by juhakaup** [comment]
I made some screen recordings when I did some variation of the design, with the idea of creating a video about the process. But I haven't had time to finish the video yet.

Adding a lot of keys to the thumb cluster might be a bit hard, as you'd have to modify the case shape quite a bit. It's not the easiest design to modify, especially if you don't have a lot of experience with fusion and even then there are some steps that are not quite obvious.

Quentin from LeBastard Keyboards has several videos about how he makes his keyboards on youtube. His process is different to what I've used, but it's still good stuff.
Another resource I could recommend is the cosmos keyboard editor https://ryanis.cool/cosmos/
